tests/__galvanic_test/configuration.rs

```diff
@@ -0,0 +1,40 @@
+use std::path::PathBuf;
+use std::{env,env::VarError};
+
+pub fn get_genesis_yaml_path() -> PathBuf {
+    let path = PathBuf::from("./tests/resources/genesis/genesis.yaml");
+    path
+}
+pub fn get_node_config_path() -> PathBuf {
+    let path = PathBuf::from("./tests/resources/jormungandr/node.config");
+    path
+} 
+
+/// Get jormungandr executable defined as system environment
+pub fn get_jormungandr_app_variable_as_path_from_os() -> PathBuf {
```

Comment:
These names could be shorter, no need to tell the whole story in the name. If you find yourself using `{common_part}_{specific_part}` names a lot, it probably means in Rust you could do with a module: `{common_part}::{specific_part}`.